include/sentry.h

```diff
@@ -1186,6 +1186,18 @@ SENTRY_API void sentry_options_add_attachment(
 SENTRY_API void sentry_options_add_attachment_n(
     sentry_options_t *opts, const char *path, size_t path_len);
 
+/**
+ * Adds a new view hierarchy attachment to be sent along.
+ *
+ * `path` is assumed to be in platform-specific filesystem path encoding.
+ * API Users on windows are encouraged to use
+ * `sentry_options_add_view_hierarchyw` instead.
```

Comment:
From https://github.com/getsentry/sentry-docs/pull/13191#pullrequestreview-2735095217

Let's extend this to say that this interface's primary use case is its use in downstream SDKs. I couldn't think of an immediate reason any direct SDK user would use this option.

Reply:
I extended it, and am also open to not adding the docs page entry for this feature.